### PR TITLE
Fix all the lint failures in the code base.

### PR DESCRIPTION
**Pull request by ericsnowcurrently**
This will allow us to start fresh and have travis check lint reliably.

### CODE REVIEW

**Comment by ericsnowcurrently** [comment]
@CanonicalLtd/landscape

File: Makefile [hunk 1]
```diff
@@ -78,16 +78,13 @@ ci-check: depends build check  ## Install dependencies and run all the tests.
 
 .PHONY: lint
 lint:
-	bzr ls-lint
+	flake8 --ignore E24,E121,E123,E125,E126,E221,E226,E266,E704 \
```
Comment (simpoir) [inline Makefile]: minor issue: at least on xenial, python3-flake8 does not provide the /usr/bin/flake8 launcher. The flake8 package provides this.

File: landscape/broker/tests/test_exchange.py [hunk 2]
```diff
@@ -677,9 +677,11 @@ def test_impending_exchange_gets_reschudeled_with_urgent_reschedule(self):
...
         # Let's make sure that that *original* impending-exchange event has
         # been cancelled:
-        self.reactor.advance(60 * 60  # time till exchange
-                             - 10  # time till notification
-                             - 20)  # time that we've already advanced
+        TIME_UNTIL_EXCHANGE = 60 * 60
+        TIME_UNTIL_NOTIFY = 10
+        TIME_ADVANCED = 20  # time that we've already advanced
+        self.reactor.advance(TIME_UNTIL_EXCHANGE -
+                             (TIME_UNTIL_NOTIFY + TIME_ADVANCED))
```
Comment (simpoir) [inline landscape/broker/tests/test_exchange.py]: ❤

File: landscape/lib/tests/test_network.py [hunk 3]
```diff
@@ -145,7 +145,7 @@ def test_get_network_traffic(self):
         self.assertEqual(traffic, test_proc_net_dev_parsed)
 
 
-#exact output of cat /proc/net/dev snapshot with line continuations for pep8
+# exact output of cat /proc/net/dev snapshot with line continuations for pep8
```
Comment (simpoir) [inline landscape/lib/tests/test_network.py]: the irony :)

File: landscape/tests/test_configuration.py [hunk 4]
```diff
@@ -1724,8 +1724,9 @@ def test_base64_ssl_public_key_is_exported_to_file(
         mock_sysvconfig().restart_landscape.return_value = True
         data_path = self.makeDir()
         config_filename = self.makeFile("[client]\ndata_path=%s" % data_path)
-        key_filename = os.path.join(data_path,
-            os.path.basename(config_filename) + ".ssl_public_key")
+        key_filename = os.path.join(
+                data_path,
+                os.path.basename(config_filename) + ".ssl_public_key")
```
Comment (simpoir) [inline landscape/tests/test_configuration.py]: overindented?

**Pull request merged**